teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md

```diff
@@ -141,7 +269,214 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowCallQueueOptOutChange
+
+When set to True users affected by the policy will be allowed to change the call queue whether to allow agent opt-out. When set to False (the default value) users affected by the policy will not be allowed to change the call queue whether to allow agent opt-out.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowCallQueueAgentOptChange
+
+When set to True users affected by the policy will be allowed to change the call queue's agent opt-in status. When set to False (the default value) users affected by the policy will not be allowed to change the call queue's agent opt-in status.
```

Comment:
... allowed to change an agent's opt-in status in the call queue.  
... will not be allowed to chagne an agent's opt-in status in the call queue.  
Note: The call queue must be configured to allow agent's to opt-out in order for this option to work.